src/api/java/appeng/api/definitions/AEItemIds.java

```diff
@@ -0,0 +1,196 @@
+    public static final ResourceLocation CALCULATION_PROCESSOR = id("calculation_processor");
+    public static final ResourceLocation ENGINEERING_PROCESSOR = id("engineering_processor");
+    public static final ResourceLocation BASIC_CARD = id("basic_card");
+    public static final ResourceLocation REDSTONE_CARD = id("redstone_card");
+    public static final ResourceLocation CAPACITY_CARD = id("capacity_card");
+    public static final ResourceLocation ADVANCED_CARD = id("advanced_card");
+    public static final ResourceLocation FUZZY_CARD = id("fuzzy_card");
+    public static final ResourceLocation SPEED_CARD = id("speed_card");
+    public static final ResourceLocation INVERTER_CARD = id("inverter_card");
+    public static final ResourceLocation SPATIAL_2_CELL_COMPONENT = id("2_cubed_spatial_cell_component");
+    public static final ResourceLocation SPATIAL_16_CELL_COMPONENT = id("16_cubed_spatial_cell_component");
+    public static final ResourceLocation SPATIAL_128_CELL_COMPONENT = id("128_cubed_spatial_cell_component");
+    public static final ResourceLocation ITEM_1K_CELL_COMPONENT = id("1k_cell_component");
+    public static final ResourceLocation ITEM_4K_CELL_COMPONENT = id("4k_cell_component");
+    public static final ResourceLocation ITEM_16K_CELL_COMPONENT = id("16k_cell_component");
+    public static final ResourceLocation ITEM_64K_CELL_COMPONENT = id("64k_cell_component");
+    public static final ResourceLocation EMPTY_STORAGE_CELL = id("empty_storage_cell");
+    public static final ResourceLocation WOODEN_GEAR = id("wooden_gear");
+    public static final ResourceLocation WIRELESS_RECEIVER = id("wireless_receiver");
+    public static final ResourceLocation WIRELESS_BOOSTER = id("wireless_booster");
+    public static final ResourceLocation FORMATION_CORE = id("formation_core");
+    public static final ResourceLocation ANNIHILATION_CORE = id("annihilation_core");
+    public static final ResourceLocation SKY_DUST = id("sky_dust");
+    public static final ResourceLocation ENDER_DUST = id("ender_dust");
+    public static final ResourceLocation SINGULARITY = id("singularity");
+    public static final ResourceLocation QUANTUM_ENTANGLED_SINGULARITY = id("quantum_entangled_singularity");
+    public static final ResourceLocation BLANK_PATTERN = id("blank_pattern");
+    public static final ResourceLocation CARD_CRAFTING = id("crafting_card");
+    public static final ResourceLocation FLUID_1K_CELL_COMPONENT = id("1k_fluid_cell_component");
+    public static final ResourceLocation FLUID_4K_CELL_COMPONENT = id("4k_fluid_cell_component");
+    public static final ResourceLocation FLUID_16K_CELL_COMPONENT = id("16k_fluid_cell_component");
+    public static final ResourceLocation FLUID_64K_CELL_COMPONENT = id("64k_fluid_cell_component");
+
+    private static ResourceLocation id(String id) {
+        return new ResourceLocation("appliedenergistics2", id);
+    }
```

Comment:
Why is there no method like this in AEBlockIds?
Or, better, a public one for both to use?

Reply:
I am not yet done with all of them ;-)
They are also unsorted.